### PR TITLE
Add types to renderActivePoints

### PR DESCRIPTION
**Pull request by PavelVanecek**
No changes here, only types. I'm trying to make sense of how the active dots work.

### CODE REVIEW

File: src/cartesian/Area.tsx [hunk 1]
```diff
@@ -66,8 +66,8 @@ interface AreaProps extends InternalAreaProps {
   tooltipType?: TooltipType;
   connectNulls?: boolean;
   // whether have dot in line
-  activeDot?: AreaDot;
-  dot?: AreaDot;
+  activeDot?: ActiveDotType;
```
Comment (ckifer) [inline src/cartesian/Area.tsx:69]: Should we not just have a dot type? dot isn't an "active" dot right?

**Review by ckifer — COMMENTED** [review]
Confused how "dot" and "activeDot" are both active

File: src/cartesian/Area.tsx (hunk 1 above)
Comment (PavelVanecek) [reply src/cartesian/Area.tsx:69]: I am also confused. The original were identical so I used the same.

Comment (ckifer) [reply src/cartesian/Area.tsx:69]: lol. Are there differences between DotProps and ActiveDotType?

Comment (PavelVanecek) [reply src/cartesian/Area.tsx:69]: `any` vs `any`

**Review by ckifer — APPROVED** [review]

**Pull request merged**